bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/testcases/ExtractionScreenTests.kt

```diff
@@ -67,4 +68,81 @@ class ExtractionScreenTests {
         idlingResource.waitForIdle()
         extractionScreen.clickTransferSummaryButton()
     }
+
+    @Test
+    fun test3_editIbanFieldAndClickSendFeedbackButton() {
+        mainScreen.clickPhotoPaymentButton()
+        onboardingScreen.clickSkipButton()
+        captureScreen.clickFilesButton()
+        captureScreen.clickPhotos()
+        imageUploader.uploadImageFromPhotos()
+        imageUploader.clickAddButton()
+        idlingResource.waitForIdle()
+        reviewScreen.assertReviewTitleIsDisplayed()
+        reviewScreen.clickProcessButton()
+        idlingResource.waitForIdle()
+        extractionScreen.editIbanField()
```

Comment:
[Request] Could this test contain an assertion as well? Perhaps to check if the iban field contains the edited text?